src/System.Transactions/tests/System.Transactions.Tests.csproj

```diff
@@ -15,6 +15,10 @@
     <Compile Include="TransactionScopeTest.cs" />
     <Compile Include="AsyncTest.cs" />
     <Compile Include="EnlistTest.cs" />
+    <Compile Include="CloneTxTests.cs" />
+    <Compile Include="LTMEnlistmentTests.cs" />
+    <Compile Include="HelperFunctions.cs" />
+    <Compile Include="TestEnlistments.cs" />
```

Comment:
I am not sure how I "fix" this. I don't know system.transactions become lowercase. Is it because when I changed to the directory in posh-get, I used lowercase? I have modified my posh-git window and now the prompt has System.Transactions correctly capitalized. Is that all I need to do?

---
In reply to: [72359008](https://github.com/dotnet/corefx/pull/10350#discussion_r72359008) [](ancestors = 72359008)